src/coreclr/jit/disasm.cpp

```diff
@@ -1229,12 +1229,12 @@ size_t CbDisassembleWithBytes(DIS* pdis, DIS::ADDR addr, const BYTE* pb, size_t
         if (fFirst)
         {
             pdis->CchFormatInstr(wz, ArrLen(wz));
-            fprintf(pfile, "%-*ls %ls\n", cchBytesMax, pwzBytes, wz);
+            fprintf(pfile, "%-*ls %S\n", cchBytesMax, pwzBytes, wz);
```

Comment:
Should the other `ls` also change?

Reply:
Also, this file is only ever compiled/run on Windows (and is probably bit-rotted). Should it even be touched?

Reply:
Oops. Missed that one.

Reply:
> Also, this file is only ever compiled/run on Windows (and is probably bit-rotted). Should it even be touched?

I would prefer to either remove the file entirely or change the format flags as appropriate.